### ISSUE TITLE
enable apk only backup on devices without root access.

### ISSUE THREAD

**Issue by bharat127** [jensstein/oandbackup#98]
On devices without root permissions, give apk only backup through standard android permissions. 

Current oandbackup (v0.2.10) is useless on phones without root access.

**Comment by jensstein** [comment]
yes, this might be a good idea. there is already some non-enabled code in the project that could be used to do this.
